libraries/botbuilder-adapter-twitter/src/twitter.ts

```diff
@@ -0,0 +1,196 @@
+import { Activity, ActivityTypes, BotAdapter, TurnContext, ConversationReference, ResourceResponse, WebRequest, WebResponse } from 'botbuilder';
+import * as Twitter from 'twitter';
+import { TwitterAdapterSettings, TwitterMessage } from './schema';
+import { retrieveBody } from './util';
+
+/**
+ * @module botbuildercommunity/adapter-twitter
+ */
+
+function createTwitterClient(settings: TwitterAdapterSettings): Twitter {
```

Comment:
Not possible to put this inside the TwitterAdapter? It could be a public function if you need to be able to mock it in tests.

Reply:
Unfortunately not with the rewire module.